getting_started/scripting/gdscript/gdscript_basics.rst

```diff
@@ -1464,62 +1464,104 @@ freed.
 Signals
 ~~~~~~~
 
-It is often desired to send a notification that something happened in an
-instance. GDScript supports creation of built-in Godot signals.
-Declaring a signal in GDScript is easy using the `signal` keyword.
+Signals are a way to send notification messages from an instance that
+other instances can listen to in a generic way. Create custom signals for
```

Comment:
I would prefer use of "node" or "object" here instead of "instance". Using "instance" might imply to the beginner that this is only something you do when instancing nodes.

Reply:
Changed to "object" in fb64a084cb334abad8e4a331704f183ddeba73bb